examples/flutter_gallery/lib/demo/animation/widgets.dart

```diff
@@ -142,7 +142,7 @@ class SectionDetailView extends StatelessWidget {
       item = new Container(
         height: 240.0,
         padding: const EdgeInsets.all(16.0),
-        child: image,
+        child: new SafeArea(top: false, bottom:false, child: image),
```

Comment:
nit: I think this would read better if there'd be one parameter per line.

Reply:
Agh missed this. Fixup (that also addresses @Hixie's comment) imminent.